src/components/pages/alerts/Alert.jsx

```diff
@@ -1,63 +1,85 @@
 import BellIcon from '../../../assets/UI_component_svg/BellIcon';
+import BellIcon2 from '../../../assets/UI_component_svg/BellIcon2';
 import ExampleAlert from '../../../assets/uploaded_blueprints/alerts-floor-plan-icon.jpg';
 
 const useStyles = makeStyles({
    alertCards: {
-      marginTop: '3vh'
+      border: '2px solid #FFFFFF',
+      borderRadius: '5px',
+      width: 'auto',
+      marginTop: '3vh',
+      "&:hover": {
+         borderColor: '#A2DAD9',
+         backgroundColor: '#F4FFFF',
+      }
    },
-   alertContent: {
+   cardContent: {
       display: 'flex',
-      flex: 1,
-      flexDirection: 'column',
-      justifyContent: 'space-between',
+      alignItems: 'center',
    },
    bellIcon: {
+      marginTop: "1vh",
+      marginRight: "1vw",
       height: '30%',
       width: 'auto',
    },
-   cardContent: {
-      height: '30vh',
+   alertContent: {
       display: 'flex',
-      flexDirection: 'row',
-      alignItems: 'flex-start',
-      overflow: 'hidden',
-      padding: '3vh',
-    },
+      alignSelf: 'center',
+      flex: 1,
+      flexDirection: 'column',
+      justifyContent: 'center',
+   },
    exampleMinView: {
-      flexShrink: 0,
-      height: '100%',
+      borderRadius: '5px',
       width: 'auto',
-      marginLeft: 'auto'
+      height: 'auto',
+      maxHeight: '15vh',
+      maxWidth: 'auto',
+      border: '1px solid lightgray',
    },
 });
 
 const Alert = () => {
    const classes = useStyles();
+   let [bellIcon, setBellIcon] = useState(<BellIcon2 />);
+
    return (
       <Card className={classes.alertCards}>
-         <CardContent className={classes.cardContent}>
-            <Box marginRight="1vw">
+         <CardContent
+            onMouseOver={() => setBellIcon(<BellIcon />)}
+            onMouseOut={() => setBellIcon(<BellIcon2 />)}
+            className={classes.cardContent}
+         >
+            <Box alignSelf="flex-start">
                <IconButton className={classes.bellIcon}>
-                  <BellIcon />
+                  {bellIcon}
                </IconButton>
             </Box>
             <Box className={classes.alertContent}>
-               <Box marginBottom="5vh">
-                  <Typography variant="subtitle1">(MMMM DD, YYYY)</Typography>
-                  <Typography variant="h5">Machine Learning Suggestion</Typography>
+               <Box>
+                  <Typography variant="subtitle2" color="textSecondary">OCTOBER 4, 2021</Typography>
+               </Box>
+               <Box marginBottom="1vh">
+                  <Typography variant="h6" color="textPrimary">AS3 Reading is currently very unhealthy</Typography>
+                  <Typography variant="h6" color="textPrimary" gutterBottom="true">Open window in Room 3</Typography>
                </Box>
                <Box>
-                  <Typography>Sensor message</Typography>
+                  <Typography variant="body1" color="textPrimary">High carbon dioxide levels detected</Typography>
```

Comment:
The typography doesn't have a close.

Reply:
Sorry I don't think I understand this comment. Do you mean the closing tag?

Reply:
Nvm I'm blind I see it now. Ignore my comment LOL